app/policies/rover_navigation_policy.rb

```diff
@@ -0,0 +1,13 @@
+class RoverNavigationPolicy < ApplicationPolicy
+  def zones?
+    user_in_admin_group?
+  end
+
+  def buildings?
+    user_in_admin_group?
+  end
+
+  def rooms?
+    user_in_admin_group?
+  end
+end
```

Comment:
Please change permissions in the RoverNavigarionPolicy from user_in_admin_group? to is_rover?

Rovers should have access to these navigation proccess.